recipe/meta.yaml

```diff
@@ -10,18 +10,20 @@ source:
   sha256: fb0785b83efe599a6a956361c0691ae1dbb5318018561af10f3e915caa0048d1
 
 build:
+  skip: true # [py>37]
   entry_points:
     - py.test-benchmark = pytest_benchmark.cli:main
     - pytest-benchmark = pytest_benchmark.cli:main
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  number: 1
+  number: 2
 
 requirements:
   host:
     - python
-    - setuptools
+    - setuptools >=30.3.0
     - wheel
     - pip
+    - py
```

Comment:
Is it needed in `host` for building the package?

Reply:
Linter was keen on it being here in this instance, but happy to remove on recommendation?